src/Donations/Components/ContactsForm.tsx

```diff
@@ -343,6 +356,27 @@ function ContactsForm({}: Props): ReactElement {
             )}
           </div>
 
+          {taxIdentificationAvail ?
+         (
+         <div className={"form-field mt-30"}>
+            <MaterialTextField
+              inputRef={register({
+                required: true,
+              })}
+              label={t("taxIdentificationNumber")}
+              variant="outlined"
+              name="tin"
+              defaultValue={contactDetails.tin}
+              data-test-id="test-tin"
+              disabled={isAuthenticated}
```

Comment:
When you are logged in and do not yet have a TIN in your user profile (as most users will not have), you cannot add one any more resulting into not being able to continue the donation process as you get an error when trying to hit the donate button to continue:

<img width="438" alt="Bildschirmfoto 2021-12-23 um 11 58 51" src="https://user-images.githubusercontent.com/1532418/147231402-bb923c03-ea5e-40c4-add6-125a12d45911.png">